application/models.py

```diff
@@ -370,31 +380,104 @@ class LandingPages(Page):
             heading="SELITE",
             help_text='Selite sijoittuu otsikon yläpuolelle. Voit jättää tämän kohdan myös tyhjäksi.',
         ),
+    ]
+
+    class Meta:
+        verbose_name = 'Banner'
+
+
+class LandingPages(Page):
+    parent_page_types = ['application.LandingPagesFolder']
+    subpage_typed = []
+
+    title_and_description_color_choices = [
+        ('BLACK', 'Black'),
+        ('WHITE', 'White'),
+    ]
+
+    top_banner = models.ForeignKey(BannerPages, null=True, blank=True, on_delete=models.SET_NULL, related_name='+', verbose_name="Top banner")
+    bottom_banner = models.ForeignKey(BannerPages, null=True, blank=True, on_delete=models.SET_NULL, related_name='+', verbose_name="Bottom banner")
+
+    hero_background_image_color_choices = [('FOG', 'Sumu'), ('ENGEL', 'Engel'), ('COPPER', 'Kupari'), ('SUOMENLINNA', 'Suomenlinna'), ]
```

Comment:
Could you please move `hero_background_image_color_choices` to the top (before or after `title_and_description_color_choices`) so that all constants would be defined before model fields? For consistency within the models.py as well.